swagger/v1/court_hearing.json

```diff
@@ -10,7 +10,12 @@
         "example": "f0a91e16-1b0e-4e1f-93fe-319dda9933e6",
         "description": "The unique identifier (UUID) of this object"
       },
-      "type": { "type": "string", "enum": "court_hearings" },
+      "type": {
+        "type": "string",
+        "example": "court_hearings",
+        "enum": ["court_hearings"],
+        "description": "The type of this object - always `court_hearings`"
+      },
```

Comment:
Swagger-UI barfs when expanding the model node without the example and description.